### PR TITLE
Add consultation missing fields

### LINKED ISSUE #1679: ICU Module: Missing field: CPK/MB

### PR DESCRIPTION
**Pull request by divyagar**
Closes #1679 #1680 #1682 #1681 #1683

### CODE REVIEW

**Comment by gigincg** [comment]
@Sandra3657 @divyagar Any blockers here? It's been stale for a while now.

**Comment by divyagar** [comment]
@gigincg Me and sandra are right now working on arike. Can anyone else take this issue?

**Comment by Apurva-Nagar** [comment]
@samholics Any idea where we have to display the information we are collecting in the inputs?

#### INPUTS:
- CPK/MB
- Operation
- Special Instruction
- Date/Size/LL
- Lines and Catheters

**Comment by samholics** [comment]
@Apurva-Nagar 
will share the inputs, but have to spend some time on it.

**Comment by Apurva-Nagar** [comment]
@samholics yeah sure, no problem

**Comment by gigincg** [comment]
@samholics Any update on this

**Comment by samholics** [comment]
@aparnacoronasafe

**Comment by Apurva-Nagar** [comment]
@gigincg @samholics I discussed it in EOD and have the required information now, will mark the PR as ready for review once all the changes are made.
Thanks.

File: src/Components/Facility/ConsultationForm.tsx [hunk 1]
```diff
@@ -475,6 +646,53 @@ export const ConsultationForm = (props: any) => {
...
+    const payload = state.form.lines.map((id: any) => {
+      const type = lineIdToString(id);
+      const other_type = id === 7 ? state.form.other_lines : "";
+      const start_date = state.form.lines_insertion_date[id];
+      const site = state.form.lines_site_level_fixation[id];
+      return {
+        start_date: lineRequired ? start_date : undefined,
+        type: lineRequired ? type : undefined,
+        site: lineRequired ? site : undefined,
+        //other_type: lineRequired ? other_type : undefined,
```
Comment (ghost) [inline src/Components/Facility/ConsultationForm.tsx]: @Apurva-Nagar can you look into it? Backend was not accepting this param, i commented and now its working

Comment (Apurva-Nagar) [reply src/Components/Facility/ConsultationForm.tsx]: @abhinandanarya06 I've made some changes to the backend and created a PR, but that PR hasn't been merged. @gigincg can you please merge this (https://github.com/coronasafe/care/pull/622)

**Comment by nihal467** [comment]
@Apurva-Nagar @abhinandanarya06 check the conflict ?

**Comment by nihal467** [comment]
@divyagar @gigincg @abhinandanarya06  test failed

spfkof

once we select the option in the lines and catheters and toggle off the lines and catheters option, the selected option remain there itself.

pfewkofe

the info page feel so filled up, and lengthy, can you rearrange it like two data's in a row than keeping each data in a row

**Comment by nihal467** [comment]
@abhinandanarya06 is it good for testing now ?

**Comment by nihal467** [comment]
seswefef

@abhinandanarya06 the slider is hard to see, can you make it visible

**Comment by nihal467** [comment]
dfwf
efwef

@abhinandanarya06 @gigincg test failed, the toggle button is off for the Date/Size/LL , Lines and Catheters and in the info page, Date/Size/LL is showing data and Line and Catheters is showing heading

**Comment by ghost** [comment]
> seswefef
> 
> @abhinandanarya06 the slider is hard to see, can you make it visible

I am unable to reproduce this issue. Can you check with refresh?

**Comment by ghost** [comment]
> dfwf efwef
> 
> @abhinandanarya06 @gigincg test failed, the toggle button is off for the Date/Size/LL , Lines and Catheters and in the info page, Date/Size/LL is showing data and Line and Catheters is showing heading

Done can you test it again?
Now info page is showing invalid for not filled Date/Size/LL. Should it as it is?

One more thing backend is accepting integer value for ETT/TT. Hence I change the slider of ett to vary on integer values.

**Comment by nihal467** [comment]
ejfwpo

@abhinandanarya06 @gigincg test failed, if we don't have input, can we hide the  heading and the field, rather than showing invalid date and heading only

**Comment by nihal467** [comment]
@abhinandanarya06 @gigincg 


spkvsdofp

test failed,

1. Once, we input data in Date/size/LL and We go to enter data in the Lines and catheters, the date of insertion are auto-filled and an error is popping showing

2. Add data validation for Operation, special instruction, and advice as they are mandatory and if this mandatory field is empty when we click on "Update consultation", auto-scroll the page to the missing field text box and under the text box, display an error message like "Required Field"

**Comment by nihal467** [comment]
sdpkvd

@abhinandanarya06 @gigincg even though the column is not mandatory, it is telling to add value to the column

**Comment by nihal467** [comment]
jwefjweff

@abhinandanarya06 @gigincg the white space under the log update button is more, can you make the adjustment ?https://deploy-preview-1720--dreamy-fermat-b5024e.netlify.app/facility/7a560e62-6aae-4293-8f1a-0ba840e3449f/patient/192fdee9-397f-49bd-833a-10f700416da8/consultation/d794ba54-da42-4238-8ba9-0b628ca60069

**Comment by nihal467** [comment]
@gigincg test approved, as the functions are working, and UI problems will be made as new issues after the end-to-end testing

**Review by gigincg — APPROVED** [review]

**Pull request merged**